src/models/genre.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (connection, DataTypes) => {
+  const schema = {
+    genre: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notNull: {
+          args: true,
+          msg: 'We need a genre name'
+        },
+        notEmpty: {
+          args: true,
+          msg: 'Insert genre name'
+        }
+      }
+    }
+  }
+  const RemoveTimeStamp = {
```

Comment:
Should leave a blank line between each definition to make for easier readability